app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt

```diff
@@ -767,14 +765,13 @@ class BrowserFragment : Fragment(), BackHandler {
             ToolbarMenu.Item.OpenInFenix -> {
                 // To not get a "Display Already Acquired" error we need to force remove the engineView here
                 swipeRefresh?.removeView(engineView as View)
-                val intent = Intent(context, IntentReceiverActivity::class.java)
-                intent.action = Intent.ACTION_VIEW
+                pendingIntent = Intent(context, IntentReceiverActivity::class.java)
```

Comment:
Since I believe this OpenInFenix lambda and onStop() can only happen from the main thread, we're probably safe from out-or-order writes to pendingIntent. This is worth checking. We haven't had to deal too much with synchronized write access yet.